cli/src/commands/working-groups/startAcceptingApplications.ts

```diff
@@ -26,21 +25,19 @@ export default class WorkingGroupsStartAcceptingApplications extends WorkingGrou
         // Lead-only gate
         await this.getRequiredLead();
 
-        const opening = await this.getApi().groupOpening(this.group, parseInt(args.wgOpeningId));
-
-        if (opening.stage.status !== OpeningStatus.WaitingToBegin) {
-            this.error('This opening is not in "Waiting To Begin" stage!', { exit: ExitCodes.InvalidInput });
-        }
+        const openingId = parseInt(args.wgOpeningId);
+        // We don't need the actual opening here, so this is just for validation purposes
```

Comment:
this comment suggests factoring out the check from getOpeningForLeadAction() into its own function  ;)

Reply:
I created an aliast method called `validateOpeningForLeadAction`, since there isn't much to factor out, but this should make the code more readable without additional comments.